test/suites/dev-frontier-template/test-pov/test-evm-over-pov2.ts

```diff
@@ -50,8 +50,8 @@ describeSuite({
                 const { result, block } = await context.createBlock(rawSigned);
 
                 log(`block.proofSize: ${block.proofSize} (successful: ${result?.successful})`);
-                expect(block.proofSize).toBeGreaterThanOrEqual(15_000);
-                expect(block.proofSize).toBeLessThanOrEqual(25_000n + emptyBlockProofSize);
+                expect(block.proofSize).toBeGreaterThanOrEqual(30_000);
```

Comment:
do these values make sense? why are they different to moonbeam ones?

Reply:
Our tests seems to fail with the original values. Reverted them in https://github.com/moondance-labs/tanssi/pull/716/commits/58db5bd6992983f6a2df28a7ee03b2569cc69a90 to new values and will be looking into resolving the issue now

Reply:
Reverted to Moonbeam's values before the update in https://github.com/moondance-labs/tanssi/pull/716/commits/e8dfc8d9c0c3b23570326143b38b5d72e3b2a742